### PR TITLE
MGDOBR-1118: fix InstanceCreation test and remove TODO

### PR DESCRIPTION
**Pull request by sjamboro**
The reaction on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/209

Remove TODO part from the InstanceCreation test suite and test each steps in the popover dialog
Rising a timeout (waiting on the READY state of bridges).

### CODE REVIEW

**Comment by sjamboro** [comment]
@kelvah I have to rise timeouts for a creation and a deletion of instances. I am not sure how the deletion part was affected by #209. can you check it?

File: mocked-api/handlers.ts [hunk 1]
```diff
@@ -940,7 +940,7 @@ const resourceStatusFlow = (
   wait: boolean,
   fail: boolean
 ): void => {
-  const waitTime = wait ? 30000 : 9000;
+  const waitTime = wait ? 30000 : 12000;
```
[the remark this answers is not in the document]
Comment (sjamboro) [reply mocked-api/handlers.ts:943]: Now both quality checks are green. Before I increase the waitTime the test was failing on the mocked version. The test running on the mocked environment fails with `expected '2 of 3 steps' but was '1 of 3 steps'`

[the remark this answers is not in the document]
Comment (sjamboro) [reply mocked-api/handlers.ts:943]: @kelvah is possible that pulling hide some step for cypress? I can confirm that I (as user) was able to see all steps. I was not able to catch all steps by Cypress.

[the remark this answers is not in the document]
Comment (kelvah) [reply mocked-api/handlers.ts:943]: I don't think there are issues between the polling overviews and cypress. I think the change to `waitTime` caused most of the trouble you faced. 

I've [opened a PR](https://github.com/sjamboro/sandbox-ui/pull/2) on top of this one to try out things. I kept only the change to the timeout inside `waitTillInstanceIsReady`  plus the timeouts around the new creation steps (`1 of 3` and `2 of 3`). I think tests [are passing fine](https://github.com/kelvah/sandbox-ui/actions/runs/3188943511/jobs/5202182511). 

Let me know what you think!

**Comment by sjamboro** [comment]
@kelvah @Srihari1192 can you review this PR?

Locally I have still problem to perform the InstanceCreation test but how you can see GHAs are green.

**Review by kelvah — APPROVED** [review]

**Pull request merged**